src/Controls/src/Core/PointerEventArgs.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace Microsoft.Maui.Controls
+{
+	/// <summary>
+	///	Event arguments used for PointerGestureRecognizer.
+	/// </summary>
+	public class PointerEventArgs : EventArgs
```

Comment:
Is this going to get some more relevant information? Not sure what that would be exactly, maybe coordinates at least?

Reply:
yep, will be adding a GetPosition

Reply:
Shane will be adding it from his SecondaryClickGesture PR